Pear release 1.11.1 docs (#169)
holepunchto/pear-docs: reference/pear/cli.md

> To dump to stdout use `-` in place of `<dir>`

> `<link>` can contain a path portion to dump a subset of the files for the Pear application. For example, if you wanted to dump only the `CHANGELOG.md` from Keet into a `dump-dir` directory, you would run:

Reviewer: ```suggestion
`<link>` can contain a path portion to dump a subset of the files for the Pear application. For example, to dump only the `CHANGELOG.md` from Keet into a `dump-dir` directory run:
```

--json      Newline delimited JSON output
```

## `pear reset [flags] <link>`

Advanced. Reset an application to initial state

> Clears application storage for a given application link. You will ask for confirmation to reset as the storage will be deleted permanently and cannot be recovered. Use with caution.

Reviewer: ```suggestion
Clears application storage for a given application link. Reset confirmation will be asked for as the storage will be deleted permanently and cannot be recovered. Use with caution.
```

Author: @davidmarkclements Applied your suggestion with a slight wording tweak here: https://github.com/holepunchto/pear-docs/pull/169/commits/f35c317e4daeff07dcfebcab6d3d3ce9fb4894a5
